jujuclient/mem.go

```diff
@@ -336,6 +354,24 @@ func (c *MemStore) CurrentModel(controller string) (string, error) {
 	return controllerModels.CurrentModel, nil
 }
 
+// PreviousModel implements ModelGetter.
```

Comment:
We could improve the documentation here.

Reply:
I followed the style. With a good IDE it's good enough...

[Screencast from 04-07-2024 15:09:10.webm](https://github.com/juju/juju/assets/172264669/6dfb8514-6146-49ae-a806-128fc54b1dd1)


Maybe having a more qualified name would be quicker, but i am not convinced yet.

[Screencast from 04-07-2024 15:10:07.webm](https://github.com/juju/juju/assets/172264669/e0e7fc10-83d9-48a0-ad77-7ae1a1043e52)

What do you suggest ?